### ISSUE TITLE
分块 GGUF 遗漏转置，以及 v1/v2 转换问题的说明

### ISSUE THREAD

**Issue by shoumenchougou** [Log-Dog012/rwkv-openvino#2]
你好，检查代码并做了本地对照测试后，有两点想反馈：

1. **`export_chunked.py` 需要同步补上转置。**

   `gguf_to_ov.py` 已经正确处理了四个 attention 方阵的转置，但 [`export_chunked.py` 第 118–121 行](https://github.com/Log-Dog012/rwkv-openvino/blob/d18ec45e9581b9d79c7c6d036b82942c48386545/scripts/export_chunked.py#L118-L121) 的 `GgufSource.get()` 仍然遗漏了。对应返回值应改为：

   `return self._to(self._dq(b + f"time_mix_{gg}.weight").T)`

2. **“v1/v2 rank 缺陷是 shoumenchougou 转换 bug”的结论不成立。**

第 0 层的 v1/v2 是不参与计算的占位张量，转换时使用 a1/a2，因此 rank 和数值相同是预期行为，不是权重损坏。我们核验的 1.5B GGUF 后续层 v1/v2 本来就是正确的 rank 64，权重也与对应 PTH 一致。

OpenVINO 对照测试中，补转置后，再从 PTH 替换 v1/v2，第 0、1 层的全部输出均完全不变。因此，请修正 README 中将其归因于 GGUF 转换 bug、要求替换 v1/v2 的说明。

## 详细的测试过程

测试基于提交 `d18ec45e9581b9d79c7c6d036b82942c48386545`。

### 漏转置的位置与修复

在 [`export_chunked.py` 第 118–121 行](https://github.com/Log-Dog012/rwkv-openvino/blob/d18ec45e9581b9d79c7c6d036b82942c48386545/scripts/export_chunked.py#L118-L121)，`GgufSource.get()` 对四个 attention 方阵直接返回反量化结果，没有转置。

这些 GGUF 张量保留的是 PTH 的原始矩阵方向，而当前前向使用 `x @ W`，需要与 PTH loader 转置后的布局一致。由于都是方阵，遗漏转置不会触发 shape 报错，但会改变计算结果。

建议修复如下：

```diff
-        # 方阵 att.{key,value,receptance,output}.weight（GGUF 同朝向，不转置）
+        # GGUF 保留 PTH 原始布局；转置后供当前前向的 x @ W 使用。
         if rest in ("att.key.weight", "att.value.weight", "att.receptance.weight", "att.output.weight"):
             gg = rest.split(".")[1]
-            return self._to(self._dq(b + f"time_mix_{gg}.weight"))
+            return self._to(self._dq(b + f"time_mix_{gg}.weight").T)
```

同一提交中的 [`gguf_to_ov.py` 第 93–97 行](https://github.com/Log-Dog012/rwkv-openvino/blob/d18ec45e9581b9d79c7c6d036b82942c48386545/scripts/gguf_to_ov.py#L93-L97) 已经通过 `orient="T"` 正确处理了这四个矩阵，两个加载路径目前不一致。

### 测试环境与结果

环境：Windows、Python 3.11.15、PyTorch 2.10.0+cpu、OpenVINO 2026.3.0、gguf 0.17.1。

模型转换为 OpenVINO 计算图后，通过 `core.compile_model(..., "CPU", ...)` 执行。GGUF 权重按社区路径反量化并转换为 FP16，再转为 FP32 计算；循环状态也使用 FP32，以隔离低精度运算带来的影响。

测试提示词：`The Eiffel Tower is located in the city of`

| 测试 | 结果 |
|---|---|
| 0.1B，遗漏四个矩阵的转置 | 输出 `../QInternalInternal...` 等杂乱文本 |
| 0.1B，补上转置 | 正常续写 ` Paris, France. It is a tower...` |
| 0.1B，补转置后再从 PTH 替换 v1/v2 | 输出不变，提示词末尾 logits 完全一致 |
| 0.1B，直接通过社区模型加载 PTH | 生成的 24 个 token 与补转置的 GGUF 路径完全一致 |
| 1.5B Q4_K_M，完整提示词经过全部 24 层 | 漏转置预测下一个 token 为 `ch`，补转置后为 ` Paris` |

0.1B 使用 `rwkv7-g1d-0.1b-20260129-ctx8192.pth` 转出的 FP16 GGUF；1.5B 使用对应 20260805 G1i checkpoint 的 Q4_K_M。

0.1B 的漏转置对照是在 `gguf_to_ov.build_state` 的正确映射基础上，撤销这四个矩阵的转置，以复现 `GgufSource.get()` 中仍存在的映射问题。

测试使用独立对照入口，正确保留每层的循环状态，并将返回状态整理为输入要求的形状。

### 关于 v1/v2 的判断

在测试的 1.5B GGUF 中，第 0 层 rank 96 的 v1/v2 是不参与该层计算的占位张量，不能仅凭它与 PTH 的 rank 64 不同，就判断文件损坏。

实际在 OpenVINO 中测试第 0、1 层时，补转置后再执行 PTH v1/v2 替换，两层的全部五个输出都逐位一致。结合此前的权重核对，目前没有发现这些文件存在所述的 v1/v2 写错问题，建议一并复核 README 的相关说明。
